### ISSUE TITLE
[Spec] Push Redfish Platform Config Protocol to UEFI spec.

### ISSUE THREAD

**Issue by nicklela** [tianocore/edk2-redfish-client#6]
Redfish Platform Config Protocol pass the Tiano design review: https://edk2.groups.io/g/devel/files/Designs/2021/0319/New%20EFI%20Protocols%20for%20edk2%20Redfish%20Implementation_conclusion.pdf
The implementation is here: https://github.com/tianocore/edk2-staging/tree/edk2-redfish-client/RedfishPkg/RedfishPlatformConfigDxe

So it meets the code-first policy and we can propose ECR to UEFI specification

**Comment by changab** [comment]
Yes, we can proceed to the next step, please just got ahead to drafting an ECR.
